serverlessworkflow/modules/ROOT/pages/index.adoc

```diff
@@ -1 +1,94 @@
-== Home page
\ No newline at end of file
+= Kogito Serverless Workflow Guides
```

Comment:
wouldn't be better write a small script to generate this doc based on what we already have? we could read the index and build it from there.

Reply:
We just need a way to extract the short description of the guide. So in a sense, it would:

1. Update all guides with meta info adding a short description of it
2. A script to fetch from every guide based on `nav.doc`
3. Create a GHA after merge on main to generate a PR with the changes if needed

It's a lot of work.

We can start small, merge this PR and then:

1. Update the PR template to remind people to add/update the cards
2. Update this file with all we have for now

In parallel, we can prioritize the automation work.

Reply:
And we already have the meta:

```asciidoc
// Metadata:
:description: Orchestration of OpenAPI based services
```

Reply:
+1